### PR TITLE
feat: add categories & type for Extension card

### PR DESCRIPTION
**Pull request by Den-Bratchikov**
**Describe** what changes this pull request brings:
This pull request adds categories & type for Extension card for Docs.

**Steps to test**
1. Open the [page](https://mdr-ci.staging.k6.io/docs/refs/pull/802/merge/extensions/getting-started/explore/)
2. Confirm that everything looks and works as expected

<details>
<summary>Screenshots</summary>

Screenshot 2022-09-12 at 12 06 30

</details>

TODO: please remove test types in `extensions.json`

**References**
[Design](https://www.figma.com/file/qOPbJuR5wWjOablZeFCQtu/K6-Website-Design?node-id=10348%3A31996)

Closes: #303

### CODE REVIEW

File: src/data/doc-extensions/extensions.json [hunk 1]
```diff
@@ -217,7 +218,8 @@
         "url": "https://github.com/dgzlopes"
       },
       "official": false,
-      "categories": ["Misc"]
+      "categories": ["Misc"],
+      "type": "Out"
```
Comment (javaducky) [inline src/data/doc-extensions/extensions.json:222]: This is not actually an output extension. For purposes of this PR, we can remove all uses of the "type" attributes for now given we see it is available.

**Review by javaducky — CHANGES_REQUESTED** [review]
Design looks good to me!

My only ask is that we remove use of the `type` attribute in `extensions.json` for the moment. I will create a follow-up PR to validate the categories and types prior to publishing the docs.

**Comment by MattDodsonEnglish** [comment]
I think it looks good! For future development, it'd be cool if there were also a way to filter by categories. 

Another thing, there's really two separate concepts, right?
- Extension _type_ is whether it's JS or Output
- Extension _category_ is for the kind of work the extension does.

Also, it'd be nice if the "misc" items had more descriptive category,

**Comment by javaducky** [comment]
> For future development, it'd be cool if there were also a way to filter by categories.

💯 to that! Figured we'd write up an issue after this is completed.

> Also, it'd be nice if the "misc" items had more descriptive category,

I'll probably be doing this myself in a followup now that we'll have this.

Also for future, would love to see those items with the `official` attribute being `true` to have a star or something to show it is a Grafana-owned extension.

File: src/data/doc-extensions/extensions.json (hunk 1 above)
Comment (Den-Bratchikov) [reply src/data/doc-extensions/extensions.json:222]: Yes, i will remove all uses of the `type` attributes in `extensions.json`.

**Review by javaducky — APPROVED** [review]

**Pull request merged**